boards/arm/stm32f411e_disco/pinmux.c

```diff
@@ -18,6 +18,15 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PA2, STM32F4_PINMUX_FUNC_PA2_USART2_TX},
 	{STM32_PIN_PA3, STM32F4_PINMUX_FUNC_PA3_USART2_RX},
 #endif	/* CONFIG_UART_STM32_PORT_2 */
+#ifdef CONFIG_PWM_STM32_2
+  {STM32_PIN_PA0, STM32F4_PINMUX_FUNC_PA0_PWM2_CH1},
+#endif /* CONFIG_PWM_STM32_2 */
+#ifdef CONFIG_PWM_STM32_3
+  {STM32_PIN_PC6, STM32F4_PINMUX_FUNC_PC6_PWM3_CH1},
+#endif /* CONFIG_PWM_STM32_3 */
+#ifdef CONFIG_PWM_STM32_4
```

Comment:
If you want to add support for PWM_4,
you'll need to add PWM_4 instance in driver/pwm/
Impacted files:
-Kconfig
-Kconfig.stm32
-pwm_stm32.c

Reply:
Yes, I have this in another branch. This should not have been added here, only added for some tests, commit in the wrong branch, sorry!